.circleci/config.yml

```diff
@@ -490,8 +530,20 @@ workflows:
             branches:
               only: /.*/
 
+      - go_cross_compile:
+          context: "OC Common"
+          requires:
+            - go_lint
+          filters:
+            tags:
+              only: /.*/
```

Comment:
so this is effectively a no-op?

Reply:
@ransomw1c not quite. You're right that this change tries to keep things as much as they used to be, only faster and with less code. However, the devil is in the details and _a lot_ of small details changed, such as harmonizing released images (all debian-based, with a lot less built-in clutter), reformulating the build dependencies for sidecars, ...